server/controllers/notes.controller.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express")
```

Comment:
Controllers are not for routing purposes. Let's use a config or routes folder to contain all the routes, that takes you to the controller and then controller calls the service for you.

Reply:
ok